pages/platform/kubernetes-k8s/label-taint-node-pools/guide.en-asia.md

```diff
@@ -0,0 +1,312 @@
+
+![Choose the billing mode](images/creating-a-cluster8-bis.png){.thumbnail}
+
+Finally, enter a name for your cluster and click on the `Send`{.action} button.
+
+![Name the cluster](images/creating-a-cluster9.png){.thumbnail}
+
+The cluster creation is now in progress. It should be available within a few minutes in your OVHcloud Control Panel.
+
+### Second node pool creation
+
+Now, your Kubernetes cluster is up and running. You can see it in your OVHcloud Control Panel.
+
+![Select your cluster](images/create-a-nodepool-1.png){.thumbnail}
+
+Click on your cluster, then on `Node pools`{.action} tab.
+
+We will create our second Node pool.
+Click on `Add a node pool`{.action} button.
+
+Then enter a name for your second node pool, `hourly` for example.
+
+![Name your second node pool](images/create-a-nodepool-2.png){.thumbnail}
+
+Select a flavor for your new node pool, we can choose "B2-7" like our other node pool.
+
+![Select a flavor for your second node pool](images/create-a-nodepool-3.png){.thumbnail}
+
+In the next step, define the size of our second node pool.
+This time, we can enable the `Autoscaling`{.action} feature.
+Define the minimum and maximum pool size in that case, 0 in minimum and 10 in maximum, for example.
+
+> [!primary]
+> By enabling anti-affinity, current and future nodes will be launched on different hypervisors (physical servers), guaranteeing higher fault tolerance.
+> 
+
+![Define a size and autoscaling for your second node pool](images/create-a-nodepool-4.png){.thumbnail}
+
+And then, choose `Hourly` billing mode for this second node pool.
+
+![Billing mode](images/create-a-nodepool-5.png){.thumbnail}
+
+The node pool creation is now in progress. It should be available within a few minutes in your OVHcloud Control Panel.
+
+The second node pool will be first in `Installing` status.
+Wait, 'till its status change to `OK`.
+
+![Status Installing](images/create-a-nodepool-6.png){.thumbnail}
+
+![Status OK](images/create-a-nodepool-7.png){.thumbnail}
+
+### Check and prepare our Node pools
+
+To deploy your application on your Kubernetes cluster, we invite you to follow our guide to [configuring default settings](https://docs.ovh.com/asia/en/kubernetes/configuring-kubectl/) for `kubectl`.
+
+When you can access to the cluster through `kubectl` command, let's display our node pools:
+
+```bash
+$ kubectl get nodepool
+NAME                                            FLAVOR   AUTO SCALED   MONTHLY BILLED   ANTI AFFINITY   DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   MIN   MAX   AGE
+hourly                                          b2-7     true          false            false           1         1         1            1           0     10    4m39s
+nodepool-cc40f90c-effb-4945-b7b9-05073725d62d   b2-7     false         true             false           3         3         3            3           0     100   164m
+```
+
+Perfect, our two node pools exists and we can see the different configuration and autoscaling mode.
```

Comment:
suggested changes: "Our two node pools now exist and we can see the different configuration and autoscaling mode."